plugins/node/instrumentation-fs/src/instrumentation.ts

```diff
@@ -127,7 +149,8 @@ export default class FsInstrumentation extends InstrumentationBase<FS> {
     original: T
   ): T {
     const instrumentation = this;
-    return <any>function (this: any, ...args: any[]) {
+    functionName = memberToDisplayName(functionName) as FMember;
+    const rv = <any>function (this: any, ...args: any[]) {
```

Comment:
What is `rv` in this context? can we give it a meaningful name?

Reply:
Short for returned value 😅 Is`patchedFunction` ok?

Reply:
`patchedFunction` is great, thanks :)